### PR TITLE
PA Personal Income Tax 2023 rules

### LINKED ISSUE #1840: Update Pennsylvania tax rules for 2022 / 23

### PR DESCRIPTION
**Pull request by JudyZ13**
Fixes #1840

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: patch
+  changes:
+    fixed:
+    - Utah taxpayer credit.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Pennsylvania 2023 income tax references.
```

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- bump: patch
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]:
```suggestion
- bump: minor
```

File: changelog_entry.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  changes:
+    fixed:
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:3]:
```suggestion
    added:
```

File: policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml [hunk 4]
```diff
@@ -7,5 +7,7 @@ metadata:
   name: pa_allegheny_county_use_tax_cap
   label: PA Alleghney County use tax cap
   reference:
-    - title: PA Form PA-40 Instructions, page 22
+    - title: 2021 PA Form PA-40 Instructions, page 22
       href: https://www.revenue.pa.gov/FormsandPublications/FormsforIndividuals/PIT/Documents/2021/2021_pa-40in.pdf#page=22
+    - title: 2023 PA Form PA-40 Instructions, page 22
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml:12]:
```suggestion
    - title: 2023 PA Form PA-40 Instructions
```

File: policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml [hunk 5]
```diff
@@ -7,5 +7,7 @@ metadata:
   name: pa_allegheny_county_use_tax_cap
   label: PA Alleghney County use tax cap
   reference:
-    - title: PA Form PA-40 Instructions, page 22
+    - title: 2021 PA Form PA-40 Instructions, page 22
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml:10]:
```suggestion
    - title: 2021 PA Form PA-40 Instructions
```

File: policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml (hunk 4 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/allegheny_county.yaml:12]: delete the page from the title whenever it is in the href hyperlink

File: policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/philadelphia_county.yaml [hunk 6]
```diff
@@ -7,6 +7,7 @@ metadata:
   name: pa_philadelphia_use_tax_cap
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/philadelphia_county.yaml:7]:
```suggestion
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/pa/tax/use_tax/higher/cap/philadelphia_county.yaml:7]: delete all the name metadata

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]
So to clarify, Pennsylvania didn't change any parameters for tax programs we covered in 2021, for 2022 or 2023?

Please link an issue if there is one

**Comment by PavelMakarchuk** [comment]
> So to clarify, Pennsylvania didn't change any parameters for tax programs we covered in 2021, for 2022 or 2023?
> 
> Please link an issue if there is one

Yes, no changes, issue linked

**Pull request merged**